README.md

```diff
@@ -37,47 +37,51 @@ The metrics reporter has the following configurations:
 ### Kafka Brokers
```

Comment:
Should this say Kafka Brokers and Controllers?

Reply:
I updated that section. I think we'll probably want further improvements to the README, it's still a bit bare bones.